cas/cas.h

```diff
@@ -293,7 +294,9 @@ struct m0_cas_op {
 	 *
 	 * Array should be non-empty.
 	 */
-	struct m0_cas_recv cg_rec;
+	struct m0_cas_recv  cg_rec;
+	/* dtm0 txr descriptor */
+	struct m0_dtm0_tx_desc *cg_txd;
```

Comment:
We cannot put bare pointers into ~XDR~ XC structures. It will be treated as uint64_t. It may work fine within a single address space but won't work between different processes. It would be better to introduce `m0_dtm0_tx_desc_is_empty` and `m0_dtm0_tx_desc_init_empty` if we want this field to be optional.

See this code for details: https://github.com/Seagate/cortx-motr/blob/3323e0a8fabbe632de04947078de0c9436439205/xcode/m0gccxml2xcode#L309

Reply:
Done.